app/routes/employment.routes.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+
+const router = express.Router();
+const employment = require('../controllers/employment.controller');
+const {
+  checkUserIdentity,
+} = require('../middlewares/authentication.middleware');
+const { isLoggedIn } = require('../middlewares/authorization.middleware');
+
+// Fetch employees of q company
+router.get('/', isLoggedIn, checkUserIdentity, employment.retrieveEmployees);
```

Comment:
checkUserIdentity middleware requires userId in the body of the request, so the middleware here is useless since you only take id in the body. Also non necessary because it is not a data-changing request